<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input_Average</class>
 <widget class="QDialog" name="Input_Average">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Input_Average{background-image: url(:/picture/bgp9.png);}</string>
  </property>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>514</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>514</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="stu_name">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>263</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Student's Name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_cou_id">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>319</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_stu_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_cou_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>377</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="stu_id">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>211</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Student's ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prompt">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>-11</y>
     <width>791</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>You can query the average scores of a student
or students in the same class or a course
by inputing the corresponding information

(You can also query all the studentsâ€™ all
courses' average score by inputing nothing)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cou_id">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>327</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Course's ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cou_name">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>379</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Course's Name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_stu_id">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>203</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="class_">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>439</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Class:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_class">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>438</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
